src/Tribe/Views/V2/View.php

```diff
@@ -303,13 +303,13 @@ public static function make_for_rest( \WP_REST_Request $request ) {
 
 		// Determine context based on the request parameters.
 		$do_not_override = [ 'event_display_mode' ];
-		$not_overrideable_params = array_intersect_key( $params, array_combine( $do_not_override, $do_not_override ) );
+		$not_overridable_params = array_intersect_key( $params, array_combine( $do_not_override, $do_not_override ) );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 118 characters

<sub>hash: 890ddf23f7d02ff2c48d5dfe1e48ddce</sub>